test/test_onnx.py

```diff
@@ -99,6 +100,17 @@ def forward(self, boxes, scores):
 
         self.run_model(Module(), [(boxes, scores)])
 
+    def test_clip_boxes_to_image(self):
+        boxes = torch.randint(10, (5, 4))
+        boxes[:, 2:] += torch.randint(500, (5, 2))
+        size = torch.randn(200, 300)
+
+        class Module(torch.nn.Module):
+            def forward(self, boxes, size):
+                return ops.boxes.clip_boxes_to_image(boxes, size.shape)
+
+        self.run_model(Module(), [(boxes, size)])
```

Comment:
can we add another test value here with different sizes, to validate that `clip_boxes_to_image` work for varying sizes and it's not exported as a constant?